library/src/main/java/com/anjlab/android/iab/v3/BillingProcessor.java

```diff
@@ -259,6 +259,20 @@ else if (responseCode == BillingClient.BillingResponseCode.ITEM_ALREADY_OWNED)
 						handleItemAlreadyOwned(purchasePayload.split(":")[1]);
 						savePurchasePayload(null);
 					}
+
+					reportBillingError(responseCode, new Throwable(billingResult.getDebugMessage()));
+
+				}
+				else if (responseCode == BillingClient.BillingResponseCode.USER_CANCELED
+						|| responseCode == BillingClient.BillingResponseCode.SERVICE_UNAVAILABLE
+						|| responseCode == BillingClient.BillingResponseCode.BILLING_UNAVAILABLE
+						|| responseCode == BillingClient.BillingResponseCode.ITEM_UNAVAILABLE
+						|| responseCode == BillingClient.BillingResponseCode.DEVELOPER_ERROR
+						|| responseCode == BillingClient.BillingResponseCode.ERROR
+						|| responseCode == BillingClient.BillingResponseCode.ITEM_ALREADY_OWNED
```

Comment:
`BillingClient.BillingResponseCode.ITEM_ALREADY_OWNED` should be removed here as it is already handled in the else branch above